Pathfinder_system_abilities_attributes.ipynb

```diff
@@ -0,0 +1,2105 @@
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "attributes.head()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Masa jakiś nowych informacji: znów słowniki"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Analiza attributes: ac"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0      {'details': '', 'value': 29}\n",
+      "1      {'details': '', 'value': 28}\n",
+      "2      {'details': '', 'value': 25}\n",
+      "3      {'details': '', 'value': 41}\n",
+      "4      {'details': '', 'value': 16}\n",
+      "                   ...             \n",
+      "360    {'details': '', 'value': 46}\n",
+      "361    {'details': '', 'value': 15}\n",
+      "362    {'details': '', 'value': 33}\n",
+      "363    {'details': '', 'value': 22}\n",
+      "364    {'details': '', 'value': 24}\n",
+      "Name: ac, Length: 1206, dtype: object\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(attributes.ac)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Czy są inne kolumny niż 'details' oraz 'vaue'?\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": "False"
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "print(\"Czy są inne kolumny niż 'details' oraz 'vaue'?\")\n",
+    "check_diff_values(pd.DataFrame(data=attributes.ac), [\"details\", \"value\"])"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "outputs": [],
+   "source": [
+    "def find_other_values(df, values):\n",
+    "    result = []\n",
+    "    for i, row in df.iterrows():\n",
+    "        for col in df:\n",
+    "            if len(row[col]) != len(values):\n",
+    "                # print(row[col])\n",
+    "                result.append(row[col])\n",
+    "                continue\n",
+    "            for val in values:\n",
+    "                if val not in row[col].keys() > 0:\n",
+    "                    result.append(row[col])\n",
+    "    return result"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "outputs": [],
+   "source": [
+    "attributes_ac_details = {i: row[\"details\"] for i, row in attributes.ac.iteritems()}"
```

Comment:
To jest chyba nie używane? A jeśli jest to przeniosłabym to w inne miejsce. Bo jest tak pomiędzy funkcjami.

Reply:
Racja
Już nie ma